### PR TITLE
[JB][SUPP-472] add regimePaymentFrequency to ArrangementRequest model

### PR DESCRIPTION
**Pull request by barnesjake**

### CODE REVIEW

File: test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala [hunk 1]
```diff
@@ -52,6 +52,7 @@ class ArrangementControllerSpec extends ItSpec {
           |	}],
           |	"channelIdentifier": "eSSTTP",
           |	"regimeType": "PAYE",
+          | "regimePaymentFrequency": "Monthly",
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: just to double check, are they definitely expecting upper case `M` and not lower case `m`?

Comment (barnesjake) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: Yeah

Comment (barnesjake) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: I think it was finalised that it's always uppercased

Comment (barnesjake) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: waiiit, we have a mix, let me check

Comment (barnesjake) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: In stubs we have `Monthly` and frontend it's `monthly`. - I don't think it matters? But we should make consistent

Comment (barnesjake) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: In stubs we have `Monthly` and frontend it's `monthly`. - I don't think it matters? But we should make consistent

Comment (AndyHWChung) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: is it the relating to the same call in the FE and stub where it's different? Consistency would be better yeah if possible

Comment (darrell-plant) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala:55]: it's bit badly formatted at the moment

Comment (barnesjake) [reply test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala]: Not sure why github is displaying it like that, it's got the correct spacing in intellij,
but I've updated to be monthly

**Review by darrell-plant — APPROVED** [review]

**Pull request merged**